### PR TITLE
Forecast Model Run Aggregation Workflow

### PR DESCRIPTION
**Pull request by mpiannucci**
On every zarr file added or removed from the nextgen-dmac bucket, the multizarr update workflow runs and updates the model run aggregation that the target zarr file belongs to, as well as the "best time series" aggregation for the target model if the target file belongs to the best time series collection.

### CODE REVIEW

File: kerchunk/kerchunk_nos_roms.py [hunk 1]
```diff
@@ -27,9 +56,51 @@ def generate_kerchunked_nc(region: str, bucket: str, key: str, dest_bucket: str,
...
+        print(f'Failed to parse model run date and hour from key {key}: {e}. Skipping...')
+        return
+
+    model_run_files = fs_read.glob(f's3://{bucket}/{model_run_glob}')
+    model_run_files = sorted(['s3://'+f for f in model_run_files])
+
+    model_run_file_count = len(model_run_files)
+    print(f'Aggregating {model_run_file_count} model files for model run {model_date} t{model_hour}z...')
+
+    # TODO: Generalize this somehow?
```
Comment (jonmjoyce) [inline kerchunk/kerchunk_nos_roms.py:80]: I think it's fine for now. We might consider a config system later to help generalize things like filenames and dimension names for each dataset, which should be modifiable by an RA type person.

**Review by jonmjoyce — APPROVED** [review]
I like the added utilities. We should consider making a list (maybe mentally for now) of what else would need to be done to operationalize as we go through development. E.g. generalizing, error handling, RA configuration, are things we're not focused on now but may be asked to estimate in the near future.

File: kerchunk/kerchunk_nos_roms.py (hunk 1 above)
Comment (mpiannucci) [reply kerchunk/kerchunk_nos_roms.py:107]: Yeah I think so too. We could get slick with it, knowing now how argo can hook up so easily to cloud filesystems. But for a prototype it works well

**Comment by mpiannucci** [comment]
Screenshot 2023-03-24 at 4 25 53 PM

**Pull request merged**